Makefile

```diff
@@ -31,22 +30,24 @@ build: | $(O) ## Build reflect binaries
 install: ## Build and install binaries in $GOBIN
 	go install -ldflags='$(GO_LDFLAGS)' $(CMDS)
 
-tiny: | $(O) ## Build and test for tinygo
-	tinygo test ./...
+tiny: | $(O) ## Build for tinygo / wasm
 	# Optimise tinygo output for size, see https://www.fermyon.com/blog/optimizing-tinygo-wasm
 	tinygo build -o $(O)/evy.wasm -target wasm -no-debug -ldflags='$(GO_LDFLAGS)'
 
 tidy: ## Tidy go modules with "go mod tidy"
 	go mod tidy
 
-.PHONY: build install tidy
+.PHONY: build install tidy tiny
 
 # --- Test ---------------------------------------------------------------------
 COVERFILE = $(O)/coverage.txt
 
-test: | $(O) ## Run tests and generate a coverage file
+test: | $(O) ## Run non-tinygo tests and generate a coverage file
 	go test -coverprofile=$(COVERFILE) ./...
 
+tinytest: | $(O) ## Run tinygo tests
```

Comment:
i wonder if the target should be `testtiny` or `test-tinygo` or something? Maybe it helps to have test targets begin with `test`

Reply:
done.